### ISSUE TITLE
Do you support adding normal samples to Sam2-Unet network training?

### ISSUE THREAD

**Issue by AaddX-ai** [WZH0120/SAM2-UNet#28]
What I mean is that I added normal samples to the training, and the mask image is all black. Will this reduce false positives and improve segmentation stability

**Comment by xiongxyowo** [comment]
Hi, for most public segmentation datasets, both the training and test sets contain only positive samples. In practice, this can lead to a higher false positive rate. Adding negative samples to the training set can help mitigate this, but it may lower the accuracy on positive samples. To address this, you might consider carefully adjusting the loss function to make the network more compatible with negative samples, as current IoU-based losses tend to be quite sensitive to all-zero black masks.

**Comment by AaddX-ai** [comment]
many thanks for u@xiongxyowo
